docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`2698` rotki will now remotely update asset symbols' mappings for different exchanges.
```

Comment:
luki is already adding it here
![image](https://github.com/rotki/rotki/assets/5068010/2d4863ea-7623-489e-b488-907d448afbc0)

Also the issue is not about remote updates but managing them right? Remove the changelog entry from here